RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -39,7 +39,9 @@ def TICL_iterations_withReco(process):
     missing_layers = 3,
     min_clusters_per_ntuplet = 5,
     min_cos_theta = 0.99, # ~10 degrees                                              
-    min_cos_pointing = 0.9
+    min_cos_pointing = 0.9,
+    out_in_dfs = True, 
+    max_out_in_hops = 10,
```

Comment:
these are identical to what's already in TrackstersProducer, no need to "modify"